scripts/rki-update.py

```diff
@@ -59,111 +56,64 @@
 fname = "berlin-cases-meldedatum.csv"
 print(fname)
 mberlin = (
-    csv
-    >> filter_by(X.Bundesland == "Berlin")
-    >> group_by(X.Bundesland, X.myear, X.mmonth, X.mday)
-    >> summarize(cases=X.AnzahlFall.sum())
-    >> arrange(X.myear, X.mmonth, X.mday)
-    >> rename(year=X.myear, month=X.mmonth, day=X.mday)
+    csv_bundeslaender
```

Comment:
this does the exact same thing to the exact same file as the code above; why are we doing it twice now?